spec/jsSpecs/ares-log.spec.js

```diff
@@ -13,23 +13,25 @@ const exec = require('child_process').exec,
 
 const aresCmd = 'ares-log',
     savedlogPath = path.join(__dirname, "..", "tempFiles", "savedlog.log"),
-    logRegExp = /\w+ \d+ \d\d:\d\d:\d\d [\w|\d]+ [\w|\d|\.|\-]+\[\d+]:/g;
+    journalLogRegExp = /\w+ \d+ \d\d:\d\d:\d\d [\w|\d]+ [\w|\d|\.|\-]+\[\d+]:/g,
+    pmlogRegExp = /\d*-\d*-\d*T\d*:\d*:\d*.\d*Z \[\d*.\d*\] \w*.\w* \w*/g;
```

Comment:
Please modify name such as :
`journaldRegExp, pmlogdRegExp.`

Reply:
It seems log format of journald and pmlogd, right?

Reply:
They refer to the log text, not log daemon. So I think the names seems appropriate.

Reply:
The reason that request to change is 
journal**Log**RegExp / pmlogRegEXp. 

"Log" keyword is in only journalLogRegExp.

Reply:
done